src/index.js

```diff
@@ -1,26 +1,44 @@
 import './style.css';
-import Selector from '../modules/selectors.js';
-import Todo from '../modules/todo.js';
 
-const objectSelector = new Selector();
-const objectTodo = new Todo();
+const todoBody = document.querySelector('.todo_main_content');
 
-window.addEventListener('load', () => {
-  objectTodo.dispaly();
-});
-objectSelector.todoForm.addEventListener('submit', (e) => {
-  objectTodo.addTask(e);
-});
+const todos = [
+  {
+    index: 1,
+    description: 'Do Laundry',
+    completed: true,
+  },
+  {
+    index: 2,
+    description: 'Go shopping',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'Clean house',
+    completed: true,
+  },
+  {
+    index: 4,
+    description: 'Walk the dog',
+    completed: false,
+  },
+];
 
-objectSelector.todoBody.addEventListener('click', (e) => {
-  objectTodo.handleFormAction(e);
-});
+const renderTodoList = () => {
+  const todoListHtml = todos.map((todo) => `
+    <li data-index="${todo.index}" data-completed="${todo.completed}">
+      <label>
+        <input type="checkbox" ${todo.completed ? 'checked' : ''} />
+        <span>${todo.description}</span>
+      </label>
+      <button class="js-delete">
+        <i class="ri-delete-bin-fill"></i>
+      </button>
+    </li>
+  `).join('');
 
-objectSelector.clear.addEventListener('click', () => {
-  objectTodo.todos = objectTodo.todos.filter((todo) => todo.status !== true);
-  objectTodo.todos.forEach((todo, id) => {
-    todo.id = id + 1;
-  });
-  localStorage.setItem('todos', JSON.stringify(objectTodo.todos));
-  objectTodo.dispaly();
-});
+  todoBody.querySelector('.js-todo-list').innerHTML = todoListHtml;
+};
+
+renderTodoList();
```

Comment:
- ![Screenshot from 2023-07-21 14-12-24](https://github.com/misikir21/my-todo-list/assets/107632780/1e390e84-a02e-4974-abe2-4c14f685be81)

- [ ] According to the project requirement, a user should be able to `add, remove, save to localStorage and edit `todo items, I can see you have the code in your codebase but looking at the video below, you can see that these functionalities are not in the UI, I'll suggest you go through your code and make sure these features are implemented correctly and working. :+1: :+1: 

- - Video reference:

https://github.com/misikir21/my-todo-list/assets/107632780/b2643c5d-3e38-41ed-a523-c3c3b9ce6bde

- [ ] Also to make the project structure more organized, I'll suggest you move the `module` folder into the `src` folder. :+1: :+1:  You also seem to have a `workflows` folder on the root of your project which is nit useful, I'll suggest you remove this too. :wave: :airplane: